org.eclipse.xtext.xtext.bootstrap/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+	compile project(':org.eclipse.xtext')
+	compile project(':org.eclipse.xtext.xtext.generator')
+	// compile "org.eclipse.xtext:org.eclipse.xtext:$versions.xtext_bootstrap"
+	// compile "org.eclipse.xtext:org.eclipse.xtext.xtext.generator:$versions.xtext_bootstrap"
```

Comment:
Depends on what we wanna have in the future.
Would we add the mwe execution into the build file?
Probably we wouldn't run it within the auto build as it requires checkouts of xtext-eclipse and xtext-idea.